modules/plug_config_example/tests/PlugConfigExampleBasicTest.test

```diff
@@ -50,17 +50,17 @@ class PlugConfigExampleBasicTest extends DrupalWebTestCase {
     $edit['name'] = 'monday';
     $this->drupalPost(NULL, $edit, 'Save Config day');
     $this->assertText('Config day Monday created');
-    $this->assertText('Monday (Machine name: monday)');
+    $this->assertText('Monday  (Machine name: monday)');
```

Comment:
Hah, tests failed because Drupal adds 2 spaces.... Here is the HTML generated:
<td>Monday <small> (Machine name: monday)</small>
We could open an issue in Entity to remove the extra space... ;)